### PR TITLE
Pull Request for project-setup : Leaderboard

### PR DESCRIPTION
**Pull request by anita00001**
## In this branch, we have:
-  Create an npm project with webpack
- Add files and setup for linters check
- Write plain HTML markup for the Leaderboard project
- Add add-score functionality
- Add functionalities for local storage
- Display the array in descending order of the score
- Deploy page using gh-pages

## Other features:
- No linters
- Professional documentation
- Use of correct GitHub-flow
- Use of Gitflow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": true,
+  "scripts": {
```
Comment (jasonfelice) [inline package.json:6]: - [ ] Currently, there's no way to start this project. Please make sure it has scripts like `start`. 😸 
_You can add the following:_
```
"start": "webpack --config webpack.config.js --watch"
```

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @anita00001, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No Linter errors✔️
Followed GitFlow✔️
Descriptive PR✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (anita00001) [reply package.json:6]: @jasonfelice Do you mean I can add the script given above? Do I need to install any other dependencies?

Comment (omar-muhamad) [reply package.json:6]: - [x] The webpack dev server still doesn't work, please follow the steps in [this section](https://webpack.js.org/guides/development/#using-webpack-dev-server) of the webpack documentation about using the webpack dev server.

[Hint] The suggestions that code reviewers give are not always right you suppose to test it first and if it doesn't work search for the solution yourself. For example, the correct script you suppose to add as mentioned in the link I gave you above is this:
```json
"start": "webpack serve --open",
```

[Hint]: While applying the dev server to your project I faced another error with the output bundle file so Kindly ensure that you change the output section in your `webpack.config.js` file to this:
```javascript
output: {
    filename: '[name].bundle.js',
    path: path.resolve(__dirname, 'dist'),
    clean: true,
 },
```

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @anita00001,

#### Nice job making the required changes. Unfortunately, you missed some of them but you are almost there.



## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: package.json (hunk 1 above)
Comment (anita00001) [reply package.json:6]: Thank you so much @Omar-Muhamad for clarifying my confusion regarding **Build using Webpack dev Server**.
I appreciate your time and effort in bringing me the solution.

**Review by prolajumokeoni — APPROVED** [review]
Hi @anita00001 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- HTML code looks like in the wireframe ✅
- Correct basic project structure 🚀

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (omar-muhamad) [reply package.json:6]: My Pleasure, 
Have a great day

**Pull request merged**